src/match/validators.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_times_list(value):
```

Comment:
by times do you mean like timestamps? or like availabilities? might be helpful to write a docstring here or sth